src/services/conditionalTokens.ts

```diff
@@ -49,6 +54,17 @@ export class ConditionalTokensService {
     }
   }
 
+  static getCollectionId(parentCollection: string, conditionId: string, indexSet: BigNumber) {
+    return CTHelpers.combineCollectionIds([
+      parentCollection,
+      CTHelpers.getCollectionId(conditionId, indexSet),
```

Comment:
The `getCollectionId` receive 3 parameters, Am I missing something here? 

`function getCollectionId (bytes32 parentCollectionId, bytes32 conditionId, uint indexSet)externalviewreturns(bytes32)`

See https://docs.gnosis.io/conditionaltokens/docs/devguide03/

Reply:
I think this was more general for nested conditions, and we can use CTHelpers off-chain provided by gnosis.